src/_langs/en/tools/index.html

```diff
@@ -16,7 +16,7 @@
   <div class="container">
     <p class="headliner">Get productive.</p>
     <div class="desc">
-      Discover our <h1>tools</h1> and kickstart your development.
+      Discover our&nbsp;<h1>tools</h1>&nbsp;and kickstart your development.
```

Comment:
I wanted "tools" to be the h1 for semantic/SEO purposes, then discovered that our minification process blows away the white-space around the tags. I know this looks ugly :/ I'd be all ears for a better suggestion.